README.md

```diff
@@ -157,7 +165,7 @@ TypeScript is more strict here, in that if you want to use a property which is n
 ```js
 type ExactUser = { name: string, age: number };
 type User = { name: string, age: number, [otherProperty: string]: any };
-type OptionalUser = Partial<{ name: string, age: number }>; // all properties become optional
+type OptionalUser = Partial<ExactUser>; // all properties become optional
```

Comment:
I wanted to be explicit here, I think `Partial<Exact>` can be confusing.

Reply:
I thought about this, and Flow example is written this way, so it maybe confuser if the examples are different.

The meaning of both expressions are the same, but they are different from what Flow expression means, I think.